plugins/funds.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+""" This plugin gives you a nicer overview of the funds that you own.
+
+Instead of calling listfunds and adding all outputs and channels
+this plugin does that for you. 
+
+Author: Rene Pickhardt (https://ln.rene-pickhardt.de)
+
+"""
+
+import json
+import sys
+
+from lightning.lightning import LightningRpc
+from os.path import expanduser
+
+path = expanduser("~/.lightning/lightning-rpc")
+
+rpc_interface = LightningRpc(path)
+
+
+def json_get_funds(request, unit="s"):
+    if unit.lower() == "bitcoin" or unit.lower() == "btc":
+        unit = "B"
+    elif unit.lower() == "satoshi" or unit.lower() == "satoshis":
+        unit = "s"
+    elif unit.lower() == "bit" or unit.lower() == "bits":
+        unit = "b"
+    elif unit.lower() == "milli" or unit.lower() == "mbtc":
+        unit = "m"
+
+    switcher = {
+        "s": 1,
+        "b": 100,
+        "m": 100*1000,
+        "B": 100*1000*1000,
+    }
+
+    div = switcher.get(unit, 1)
+
+    switcher = {
+        "s": "sat",
+        "b": "bit",
+        "m": "mBTC",
+        "B": "BTC",
+    }
+    unit = switcher.get(unit, "sat")
+
+    info = rpc_interface.getinfo()
+    funds = rpc_interface.listfunds()
+    outputs = funds["outputs"]
+    onchain_value = 0
+    for output in outputs:
+        onchain_value += int(output["value"])
```

Comment:
Nothing wrong with this code, but fyi, you can also use list comprehensions to shorten this down a bit to something like:
```
onchain_value = sum([int(output['value']) for output in funds['outputs']])
offchain_value = sum([int(channel["channel_sat"]) for channel in funds['channels']])
```

Reply:
yeah I will do that since I have to touch the code once more anyway (: